spec/beaker/host/unix/file_spec.rb

```diff
@@ -196,6 +196,28 @@ def logger
       end
     end
 
+    describe '#chmod' do
+      context 'recursive' do
+        it 'calls execute with chmod' do
+          path = '/path/to/file'
+          mod = '+x'
+
+          expect( instance ).to receive(:execute).with("chmod #{mod} #{path}")
+          instance.chmod(mod, path)
+        end
+      end
+
+      context 'not recursive' do
+        it 'calls execute with chmod' do
+          path = '/path/to/file'
+          mod = '+x'
+
+          expect( instance ).to receive(:execute).with("chmod -R #{mod} #{path}")
+          instance.chmod(mod, path, true)
+        end
+      end
```

Comment:
the contexts are switched, first should be `not recursive` and second `recursive` as per test expectation